src/schemas/validation/schema.yaml

```diff
@@ -193,7 +193,7 @@ $defs:
         additionalProperties:
           $ref: '#/$defs/path-item'
     patternProperties:
-      '^(schemas|responses|parameters|examples|requestBodies|headers|securitySchemes|links|callbacks|pathItems)$':
+      '^(?:schemas|responses|parameters|examples|requestBodies|headers|securitySchemes|links|callbacks|pathItems)$':
```

Comment:
Same question: what is the benefit of a non-capturing pattern? I find it confusing.